src/main/kotlin/dto/CoordinateGeneratorRequest.kt

```diff
@@ -0,0 +1,14 @@
+package dto
+
+import domain.Coordinate
+import util.ConvertType
+
+class CoordinateGeneratorRequest(val coordinates: List<Coordinate>, val landMine: Int) {
+
+    companion object {
+        fun of(landMineMapRequest: LandMineMapRequest, landMine: String): CoordinateGeneratorRequest {
+            val coordinates: List<Coordinate> = landMineMapRequest.height.flatMap { y -> landMineMapRequest.width.map { x -> Coordinate(x, y) } }
```

Comment:
연속된 메서드 호출을 한다면 개행을 해주는 것이 좋습니다.
비단 이 부분 뿐만 아니라 다른 곳도 찾아서 동일하게 개선해주세요.

https://kotlinlang.org/docs/coding-conventions.html#wrap-chained-calls

```suggestion
            val coordinates: List<Coordinate> = landMineMapRequest.height
                .flatMap { y -> landMineMapRequest.width.map { x -> Coordinate(x, y) } }
```

Reply:
추가로 map 내부에서 또 map을 호출하는 것은 indent 가 늘어난 것으로 보입니다. 프로그래밍 요구사항을 지켜서 개선해주세요.